src/frontend/src/flows/manage/authenticatorsSection.ts

```diff
@@ -76,15 +76,18 @@ export const authenticatorsSection = ({
           </span>
         </div>
         ${
-          warnFewDevices
+          warnNoPasskeys
```

Comment:
ummm would it be cleaner to have a single `html` parser and the condition within the string?

```
html`<p
                style="max-width: 30rem;"
                ${warnNoPasskeys ? class="warning-message t-paragraph t-lead" : ''}
              >
              ...
```
              
what do you think?